.circleci/config.yml

```diff
@@ -26,17 +26,18 @@ aliases:
 defaults: &defaults
   working_directory: ~/buie
   docker:
-    - image: cypress/base:14.19.0
+    - image: cimg/node:18.16.0
```

Comment:
updated the general image for over tasks, this image does not include the env or dependencies for cypress but is immediately available on circleci without downloading it.

https://circleci.com/developer/images/image/cimg/node